user-console/src/main/resources/org/pentaho/mantle/public/themes/ruby/mantleRuby.css

```diff
@@ -1054,18 +1054,69 @@ div#customDropdownPopupMajor .gwt-MenuBar-vertical .gwt-MenuItem:hover:not(.gwt-
 }
 
 #generated-content-location {
-  width: 350px;
+  /*width: 350px;*/
   border-radius: 4px 0 0 4px;
 }
 
 .schedule-label {
-  margin-top: 15px;
-  margin-bottom: 5px;
+  margin-top: unset;
+  margin-bottom: unset;
   color: #666666;
 }
 
+.new-schedule-dialog .schedule-dialog-content > table > tbody {
+  gap: var(--dialog-vertical-gap);
+}
```

Comment:
Should not be needed anymore.

Reply:
Removing this statement is making the gap disappear entirely.